src/src/hw9/hillel/ArrayValueCalculator.java

```diff
@@ -0,0 +1,40 @@
+package hw9.hillel;
+
+public class ArrayValueCalculator {
+
+    public static int doCalc(String[][] arr) throws ArraySizeException, ArrayDataException {
```

Comment:
`throws ArraySizeException, ArrayDataException` should be used if you wanna delegate error handling to the end user. The description asks you to validate the input on your own, so throws is redundant here, remove it.